Lib/concurrent/futures/process.py

```diff
@@ -855,3 +856,31 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def terminate_workers(self, signal=signal.SIGTERM):
```

Comment:
Actually, can we perhaps name it `kill`? (namely, you kill the pool's workers with the given signal) or is `kill` already taken (possibly for something else in the future)?

Reply:
I'm not sure. I thought this was explicit. I'm open for either or.

Reply:
Ok, so after some research:

- `Process.terminate` -- mimics `SIGTERM`
- `Process.kill` -- sends `SIGKILL` to the process.

Many other interfaces use `terminate` for sending SIGTERM and `kill` for sending `SIGKILL`. Now, we also have [multiprocessing.Pool.terminate](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.Pool.terminate), so maybe we could mimic it by just naming it `terminate(signal=...)`? It would be slightly inconsistent with the others `terminate` methods since this one would be able to send a SIGKILL as well.

So it's up to you (or up to Gregory)

Reply:
I could definitely be down for `terminate` to match multiprocessing.Pool. (Though i'd still like to be able to change the signal sent so it would be a slight api difference).  I'll wait to see what Gregory thinks to either change it or leave as is.